### PR TITLE
Fix broken export because of $selectFieldDefinitionKey is undefined

### PR DESCRIPTION
**Pull request by nestordedios**

### CODE REVIEW

**Comment by nestordedios** [comment]
This fixes #29 .

@I-Valchev  issue is fixed and it can now be merged by @bobdenotter .

👍

**Review by I-Valchev — APPROVED** [review]
👍 Awesome, thanks @nestordedios!

File: src/Export.php [hunk 1]
```diff
@@ -141,10 +141,10 @@ private function updateSelectFields(array $currentITem, $selectFields)
         foreach ($selectFields as $selectFieldDefinitionKey => $selectFieldDefinition) {
             $selectFieldData = $currentITem['fields'][$selectFieldDefinitionKey];
             $data = $this->populateSelectFieldReferencedData($selectFieldData, $selectFieldDefinitionKey, $selectFieldDefinition);
-        }
 
-        // Update the reference of the imported select field value.
-        $currentITem['fields'][$selectFieldDefinitionKey] = $data;
+            // Update the reference of the imported select field value.
+            $currentITem['fields'][$selectFieldDefinitionKey] = $data;
```
Comment (bobdenotter) [inline src/Export.php:146]: Let's fix the capitalisation of this (and also in the method definition, on line 139) 

```suggestion
            $currentItem['fields'][$selectFieldDefinitionKey] = $data;
```

Comment (nestordedios) [reply src/Export.php]: Good catch @bobdenotter. 

It's fixed in https://github.com/bobdenotter/conimex/pull/28/commits/c625c7fc3d783819e984017c218ea92c7d78b65d

File: src/Export.php [hunk 2]
```diff
@@ -141,10 +141,10 @@ private function updateSelectFields(array $currentITem, $selectFields)
...
+        }
 
         return $currentITem;
```
[the remark this answers is not in the document]
Comment (nestordedios) [reply src/Export.php]: Also fixed in https://github.com/bobdenotter/conimex/pull/28/commits/c625c7fc3d783819e984017c218ea92c7d78b65d

**Pull request merged**